packages/vuelidate/src/index.js

```diff
@@ -42,9 +42,9 @@ export function useVuelidate (validations, state, globalConfig = {}) {
   // if there is no registration name, add one.
   if (!$registerAs && instance) {
     // NOTE:
-    // ._uid // Vue 2.x Composition-API plugin
+    // .proxy._uid // Vue 2.7
     // .uid // Vue 3.0
-    const uid = instance.uid || instance._uid
+    const uid = instance.uid || instance.proxy._uid
```

Comment:
I think this was recently changed by a PR. Can you check it?

Reply:
You mean merged PR ? I haven't seen it - this branch is based on the latest `next`